src/components/FeatureViews/Accounts/Accounts.js

```diff
@@ -198,7 +198,10 @@ const Accounts = () => {
                         </TableCell>
                         <TableCell>
                             <Tooltip title={row.user.email}>
-                                <span>{row.user.email.substr(0, 20)}</span>
+                                <span>
+                                    {row.user.email.substr(0, 20)}
+                                    {(row.user.email.length > 21) && "..."}
```

Comment:
2 things:
1. avoid having magic numbers - define them in a const variables (remember to use all caps, example: MAX_EMAIL_LENGTH) 
2. use an explainer variable for your condition. could be something like: `isOverMaxEmailLength`